.goreleaser.yml

```diff
@@ -25,9 +25,6 @@ checksum:
   algorithm: sha256
 
 release:
-<<<<<<< HEAD
-  name_template: "{{.Version}} (WARNING: BETA SOFTWARE)"
```

Comment:
This is the only hunk in the original PR (#9371). Are the changes shown below intended?

Reply:
They were intended. It's not 100% clear to me whether Goreleaser will be archiving the root docs on the release branch (i.e. `v0.34.x`) when cutting new patch releases, or the docs on `main`, so I thought I'd use this opportunity to sync the two sets of docs.

See https://github.com/tendermint/tendermint/pull/9373/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689R31

Reply:
I see